packages/resolve-scripts/src/runtime/client/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { render } from 'react-dom'
+import { Provider } from 'react-redux'
+import createHistory from 'history/createBrowserHistory'
+import { ConnectedRouter } from 'react-router-redux'
+
+import Routes from './components/Routes'
+import createStore from './store/create_store'
+
+const routes = require($resolve.routes)
+
+const history = createHistory({
+  basename: process.env.ROOT_PATH
+})
+const store = createStore(window.__INITIAL_STATE__, history)
```

Comment:
Does `InitialState` exist anymore? Nor read-models or view-models uses some *default* initial state, nor application have some *default* state without relation to some read or view model